ArduPlane/altitude.cpp

```diff
@@ -359,9 +359,9 @@ void Plane::check_fbwb_altitude(void)
     }
 #endif
 
-    if (g.FBWB_min_altitude_cm != 0) {
```

Comment:
This is a change in behavior. I guess setting a negative altitude is a bit of a edge case, especially if 0 is a magic disable.